app/src/main/java/org/dhis2/usescases/teiDashboard/TeiDashboardPageConfigurator.kt

```diff
@@ -3,11 +3,12 @@ package org.dhis2.usescases.teiDashboard
 import org.dhis2.utils.customviews.navigationbar.NavigationPageConfigurator
 
 class TeiDashboardPageConfigurator(
-    val dashboardRepository: DashboardRepository
+    val dashboardRepository: DashboardRepository,
+    val isPortrait: Boolean
 ) : NavigationPageConfigurator {
 
     override fun displayDetails(): Boolean {
-        return true
+        return isPortrait
```

Comment:
I don't understand the objective of the function, displayDetails() returns a boolean but is not a question, maybe you could change the name to a more descriptive function name.

Reply:
This class overrides the NavigationPageConfigurator that have all those methods to handle the bar sections visibility